metadata_service/proxy/neptune_proxy.py

```diff
@@ -57,7 +57,7 @@ class NeptuneGremlinProxy(AbstractGremlinProxy):
     def __init__(self, *, host: str, port: Optional[int] = None, user: str = None,
                  password: Optional[Union[str, boto3.session.Session]] = None,
                  driver_remote_connection_options: Mapping[str, Any] = {},
-                 neptune_bulk_loader_s3_bucket_name: Optional[str] = None,
```

Comment:
There was no way to define this thru configuration. Decided to reuse the existing client_kwargs field.

Reply:
Oh, it's a little magical, but if you do like follows:
```
options = current_app.config[config.PROXY_CLIENT_KWARGS] or {}
_proxy_client = NeptuneGremlinProxy(host=host, port=port, user=user, password=password, **options)
```
python should pick up any keys from the PROXY_CLIENT_KWARGS dict and use them as keyword args. So that's how `neptune_bulk_loader_s3_bucket_name` would be populated here.

Reply:
are you suggesting I make those changes here: https://github.com/amundsen-io/amundsenmetadatalibrary/blob/master/metadata_service/proxy/__init__.py#L41?

Reply:
Oh, I see, the existing proxies don't use this (we've diverged a bit). It's probably least disruptive to just conform to the existing interface then, sure!